openmc/deplete/abc.py

```diff
@@ -629,6 +630,27 @@ def __init__(
             else:
                 raise ValueError(f"Invalid timestep unit '{unit}'")
 
+            # validate existing depletion steps are consistent with those passed to operator
+            if continue_timesteps:
+                completed_timesteps = operator.prev_res.get_times()
+                completed_source_rates = operator.prev_res.get_source_rates()
```

Comment:
I'm realizing now that I don't think `openmc.deplete.Results` has a `get_source_rates()` function. It seems like individual operators have a `source_rates` member, but I'm wondering if a method in `openmc.deplete.Results` parallel to `get_times()` would be useful for others. I noticed `openmc.deplete.abc.Integrator` has an `__iter__()` method but it doesn't return _all_ the source rates, just the source rate and time (for a given iteration?)

It seems like maybe the appropriate line should be 
```
completed_source_rates = operator.source_rates
```
though I'm wondering if the length of `get_times()` and `operator.source_rates` might have different lengths.

Tests were passing because I never added a test that hits this code, I will push up a test very soon (and likely it will fail)

Reply:
Implemented but we should (in a future PR) address the current lack of exporting units with `source_rates`. See #3281